core/vm/actor/storage_power_actor.cpp

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/actor/storage_power_actor.hpp"
+#include <power/impl/power_table_impl.hpp>
+#include "vm/indices/indices.hpp"
+
+namespace fc::vm::actor {
+
+  const power::Power StoragePowerActor::kMinMinerSizeStor =
+      100 * (primitives::BigInt(1) << 40);
```

Comment:
Magic (numbers) outside Hogwarts.

Reply:
What's Stor btw?

Reply:
Number and name from spec

Reply:
Then please at least comment in the code so that it's clear why exactly this number is here.